netctl/commands.go

```diff
@@ -360,12 +360,14 @@ var Commands = []cli.Command{
 					cli.StringFlag{
 						Name:  "vlan-range, v",
 						Usage: "Allowed Vlan id range",
-						Value: "1-4094",
```

Comment:
from now we can set individual parameters of the global settings. If we allow this then we will have defaults set. For example if we already had vlans 1-2000 as the global setting and i want to only change fwd mode and retain the vlans and vxlans - i should be able to do it without vlans being defaulted.